frontend/lib/views/truck_detail.dart

```diff
@@ -93,8 +94,12 @@ class _TruckDetailState extends State<TruckDetailView> {
                 child: Semantics(
                     label: _isFavorite ? 'Remove From Favorites': 'Add To Favorites',
                     button: true,
-                    child:Padding(padding: EdgeInsets.only(left: 10, top: 10, bottom: 10), 
-                      child: Image.asset(_isFavorite?'images/icon-star-selected.png':'images/icon-star.png')))
+                    child:Padding(padding: EdgeInsets.only(left: 10, top: 10, bottom: 10),
+                        child: IconButton(
+     icon: _isFavorite? Icon(Icons.favorite):Icon(Icons.favorite_border),
+     color: Colors.pink,
+     onPressed: _toggleFavTruck,
+     )))
```

Comment:
I think it would be better to be consistent with the existing app so could we change the icon back to star?